packages/sdk/src/client/sam/contract.ts

```diff
@@ -112,8 +114,7 @@ export async function SamBuy(
   { editionAddress }: SamEditionAddress,
   {
     quantity,
-
```

Comment:
in our regular non-sam mint, we pass in sale schedule and quantity, and it calculates totalValue

here, price is confusing because you may think it means price per nft, since quantity is also an arg

clarifying it is maxTotalValue (calculated with the flat tx fee, and not have the percent platform fee for new sam)